static/app/components/organizations/timeRangeSelector/index.tsx

```diff
@@ -119,6 +119,11 @@ type Props = WithRouterProps & {
    */
   detached?: boolean;
 
+  /**
+   * Aligns dropdown menu to left or right of button
+   */
+  forceAlignment?: 'left' | 'right';
```

Comment:
Why do we call it `force`?

Reply:
yea I figured since there is already a default behavior for alignment which switches between left/right, then if we want to force it to one side it would be called `forceAlignment` but `alignDropdown` is simpler. I'll switch it real quick.